consensus/scheme/rolldpos/rolldpos_test.go

```diff
@@ -416,31 +416,6 @@ func TestRollDPoSConsensus(t *testing.T) {
 			sf, err := factory.NewFactory(cfg, factory.InMemTrieOption())
 			require.NoError(t, err)
 			require.NoError(t, sf.Start(ctx))
-			for j := 0; j < numNodes; j++ {
-				ws, err := sf.NewWorkingSet()
-				require.NoError(t, err)
-				_, err = accountutil.LoadOrCreateAccount(ws, chainRawAddrs[j])
-				require.NoError(t, err)
```

Comment:
It doesn't hurt running unit test and since this is testing for rolldpos consensus, is it necessary to create block producer's account in here? I'm not sure